src/app/queries/fetchHelpers.ts

```diff
@@ -95,7 +94,6 @@ export const authorizedK8sRequest = async <T>(
     if (isAxiosSelfSignedCertError(error)) {
       const url = `${VIRT_META.clusterApi}/.well-known/oauth-authorization-server`;
       setSelfSignedCertUrl(url);
```

Comment:
Yeah, calling `setSelfSignedCertUrl` does nothing if we don't have the CertErrorPage. We should remove it from the FetchContext as well I think.